packages/graphiql-plugin-explorer/src/graphiql-explorer.d.ts

```diff
@@ -47,10 +47,10 @@ declare module 'graphiql-explorer' {
     arrowClosed?: ReactNode | null;
     checkboxChecked?: ReactNode | null;
     checkboxUnchecked?: ReactNode | null;
-    styles?: ?{
-      explorerActionsStyle?: { [key: string]: any };
-      buttonStyle?: { [key: string]: any };
-      actionButtonStyle?: { [key: string]: any };
+    styles?: {
+      explorerActionsStyle?: CSSProperties;
+      buttonStyle?: CSSProperties;
+      actionButtonStyle?: CSSProperties;
     };
```

Comment:
We should also allow `null` here to align with the flow type definitions from the `graphiql-explorer` package.
```suggestion
    styles?: {
      explorerActionsStyle?: CSSProperties;
      buttonStyle?: CSSProperties;
      actionButtonStyle?: CSSProperties;
    } | null;
```

Reply:
Yup, definitely. I totally missed this.